meson.build

```diff
@@ -0,0 +1,182 @@
+project(
+  'cloud-init',
+  meson_version: '>=0.63.0', # rockylinux/9
+  license: 'GPL-3 or Apache-2.0',
+  default_options: [
+    # The default can yield broken results.
+    'python.install_env=auto',
+  ],
+)
+system = host_machine.system()
+sysconfdir = get_option('sysconfdir')
+INIT_SYSTEM = get_option('init_system')
+
+lib_exec_dir = join_paths('/usr', get_option('libexecdir'), 'cloud-init')
```

Comment:
I don't think that we should hardcode absolute paths. Doing so defeats the flexibility of the build system and the ability for downstreams to override when needed.

The prefix option probably makes sense here.